jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/Server.java

```diff
@@ -276,6 +277,22 @@ public InvocationType getInvocationType()
         return type;
     }
 
+    public boolean isBindEarly()
+    {
+        return _bindEarly;
+    }
+
+    /**
+     * Allows to disable early binding of network sockets. Network sockets are bound early by default.
+     * @param bindEarly If {@code bindEarly} is {@code true} (default), network sockets are bound before
+     * starting other components. If {@code bindEarly} is {@code false}, network connectors open sockets
+     * when they're started.
+     */
+    public void setBindEarly(boolean bindEarly)
+    {
+        _bindEarly = bindEarly;
+    }
```

Comment:
Whilst "bind" is technically the right name, it is the `Connector.open()` method that we are calling early.
So a better name would be `setOpenEarly`  or perhaps `setOpenBeforeStart` with the description saying that the connectors are opened before any server components (including webapplications) are started.

Reply:
I've adjusted the name to `setOpenEarly`.